examples/oriented_images.js

```diff
@@ -0,0 +1,100 @@
+/* global itowns, document, renderer, initPicture  */
+// # Oriented picture
+
+// Define initial camera position. Note that camera initial position will be overwritten by picture position later
+var positionOnGlobe = {
+    longitude: 0,
+    latitude: 45,
+    altitude: 100000000 };
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+    renderer: renderer,
+    sseSubdivisionThreshold: 6,
+    noControls: true,
+});
+
+var pictureInfos;
+var coord;
+var rotationMatrix;
+// eslint-disable-next-line no-unused-vars
+var plane;
+
+function addLayerCb(layer) {
+    return globeView.addLayer(layer);
+}
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+itowns.proj4.defs('EPSG:3946',
+    '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+// Add one imagery layer to the scene
+// This layer is defined in a json file but it could be defined as a plain js
+// object. See Layer* for more info.
+itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb);
+itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb);
+
+pictureInfos = {
+    panoramic: {
+        latitude: 45.9228208442959,
+        height: 4680.55588294683,
+        longitude: 6.83256920100156,
+        azimuth: 526.3900321920207,
+        roll: 2.1876227239518276,
+        tilt: -11.668910605126001,
+        image: './orientedImages/MontBlanc.jpg',
+    },
+    camera: {
+        size: [6490, 4408],
+        focale: 1879.564256099287 * 4.4,
+        ppaX: 3245,
```

Comment:
Where does this come from as well? (I suppose that 6490 and 4408 is the image size in pixels?)

Reply:
But where does this magic `4.4` value comes from?
And how did you determine the position of the ppa? Is this always the center of the image?